napari/components/add_layers_mixin.py

```diff
@@ -896,3 +896,56 @@ def _add_layer_from_data(
                 raise exc
 
         return layer
+
+    def save_layers(
+        self,
+        path: str,
+        selected: bool = True,
+        plugin_name: Optional[str] = None,
+    ):
+        """Save all or only selected layers to a path using writer plugins.
+
+        If no `plugin_name` is provided and only one layer is targted, then we
+        just directly call ``plugin_manager.hook.napari_write_<layer>()`` which
+        will loop through implementations and stop when the first one returns a
+        non-None result. The order in which implementations are called can be
+        changed with the implementation sorter/disabler.
+
+        If no `plugin_name` is provided and multple layers are targted, then
```

Comment:
```suggestion
        If no `plugin_name` is provided and multiple layers are targeted, then
```

Having issues with the butterfly keyboard, @sofroniewn? :joy:

Reply:
done